frontend/packages/pydantic-forms/src/components/form/Footer.tsx

```diff
@@ -48,63 +54,57 @@ const Footer = () => {
-                                    className="d-flex mv-0 mr-3"
-                                    style={{ opacity: 0.8 }}
-                                >
-                                    Het formulier is nog niet aangepast
-                                </div>
-                            )}
-                    </div>
+                {rhf.formState.isValid &&
+                    !allowUntouchedSubmit &&
+                    !rhf.formState.isDirty && (
+                        <div
+                            className="d-flex mv-0 mr-3"
+                            style={{ opacity: 0.8 }}
+                        >
+                            Het formulier is nog niet aangepast
+                        </div>
+                    )}
 
-                    {!!onCancel &&
-                        (cancelButton ?? (
-                            <button
-                                type="button"
-                                onClick={onCancel}
-                                e2e-id="pydanticforms-cancel-btn"
-                                style={{
-                                    marginLeft: '8px',
-                                    height: '28px',
-                                    padding: '4px',
-                                }}
-                            >
-                                Annuleren
-                            </button>
-                        ))}
+                {!!onCancel &&
+                    (cancelButton ?? (
+                        <button
+                            type="button"
+                            onClick={onCancel}
+                            e2e-id="pydanticforms-cancel-btn"
+                            style={{
+                                marginLeft: '8px',
+                                height: '28px',
+                                padding: '4px',
+                            }}
+                        >
+                            Annuleren
+                        </button>
+                    ))}
 
-                    <button
-                        e2e-id="pydanticforms-send-btn"
-                        type="submit"
-                        style={{
-                            marginLeft: '8px',
-                            height: '28px',
-                            padding: '4px',
-                        }}
-                        disabled={
-                            !rhf.formState.isValid ||
-                            (!allowUntouchedSubmit &&
-                                !rhf.formState.isDirty &&
-                                !rhf.formState.isSubmitting)
-                        }
-                    >
-                        {sendLabel ?? 'Verzenden'}
-                    </button>
-                </div>
-            </div>{' '}
+                <button
+                    e2e-id="pydanticforms-send-btn"
+                    type="submit"
+                    style={{
+                        marginLeft: '8px',
```

Comment:
Maybe for another ticket, but It would be nice if we make our own sizes/colors file where we keep everything there.